coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentControllerBase.java

```diff
@@ -354,7 +355,7 @@ protected ModelAndView navigate(ProposalDevelopmentDocumentForm form, BindingRes
          boolean canEdit = form.isCanEditView();
          if (isNavigateAwayFromAttachment(navigateToPageId, form.getPageId())) {
              prepareLocks(form);
-             return narrativePageSave(form, canEdit);
+             return narrativePageSaveOnNavigate(form, canEdit);
```

Comment:
My concern is navigate and save should be exactly the same except the page that loads afterwards. So if this is a problem in one case it should be in both. I see the difference between the 2 methods, but in either case you are copying attachments from the form to a db loaded version, its just a question of which ones. So are ones that haven't been edited losing their attachment because of this copy or something else? And if so, why doesn't it occur when they click save?